src/Client/Web/Bit.Client.Web.BlazorUI/Components/TextField/BitTextField.Fluent.scss

```diff
@@ -86,7 +86,11 @@
     border: rem(1px) solid $Grey130;
     box-sizing: border-box;
     background-color: $White;
-    cursor: text;
+    cursor: text !important;
```

Comment:
This `!important` is for overriding the cursor style applied by **BitDatePicker** component styles
    
    .bit-txt-enabled-fluent div div {
      cursor: pointer;  // <==
      color: #605E5C;
    }